packages/vue3/src/components/Card/Card.vue

```diff
@@ -0,0 +1,54 @@
+<script lang="ts" setup>
+import { useSlots } from 'vue';
+
+const slots = useSlots();
+</script>
+
+<template>
+  <div class="card">
+    <div v-if="slots.header" class="card-header">
```

Comment:
We tend to not encourage BEM as a convention as it creates clutter.
We opt to scoped styling which will help us use shorter class names while maintaining scoped styling.
For example: 
```suggestion
    <div v-if="slots.header" class="header">
```

Then below you can simply use `.header` as your selector.
The same goes for SASS as it should only be used when we're dealing with complex styling rules, an example of that would be the `BaseButton` component, sass there is used to compose complex styles.